src/python/pants/backend/helm/goals/lint.py

```diff
@@ -40,7 +40,7 @@ async def run_helm_lint(request: HelmLintRequest, helm_subsystem: HelmSubsystem)
     def create_process(chart: HelmChart, field_set: HelmLintFieldSet) -> HelmProcess:
         argv = ["lint", chart.path]
 
-        strict = field_set.lint_strict or helm_subsystem.lint_strict
+        strict: bool = field_set.lint_strict.value or helm_subsystem.lint_strict
```

Comment:
FYI we prefer to use `cast` for reasons explained at: https://www.pantsbuild.org/v2.10/docs/style-guide#prefer-cast-to-override-annotations

No need to open a dedicated PR to fix this, it's not a very big deal. But it would be good to fix in other Helm PRs you might put up.

Reply:
ah right, I thought that the type annotation here would actually help enforce I wanted a boolean and not anything else in this assignment.

Will take into account.